src/Dotnet.Script.Tests/ScriptExecutionTests.cs

```diff
@@ -259,14 +261,6 @@ public void ShouldExecuteRemoteScript(string url, string output)
             Assert.Contains(output, result.Output);
         }
 
-        [Fact]
-        public void ShouldExecuteRemoteScriptUsingTinyUrl()
```

Comment:
tinyurl implemented captcha using cloudflare and we can't use it anymore 😅

however I am not sure why this test even exists, I assume to test if would follow redirects? if that is the case, then I added an `http://` variant to the test above, since github returns 301 then